src/libuv/libuv-windows.nuspec

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package>
+    <metadata>
+        <id>dnx-libuv-windows</id>
```

Comment:
Alright, I don't care much about the package ID then. I still recommend changing to something like `Microsoft.AspNet.Internal.libuv` in order to ensure if it ever does appear anywhere else, it is clear that it is built by Microsoft and for internal use by ASP.NET. But if no one else likes that name, meh.